generate/sre-authorization/blacklist.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "apiGroup": "rbac.authorization.k8s.io",
+        "resources": ["clusterrolebindings","rolebindings"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machineconfiguration.openshift.io",
+        "resources": ["machineconfigs","machineconfigpools","controllerconfigs"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machine.openshift.io",
+        "resources": ["machines","machinesets"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "config.openshift.io",
+        "resources": ["oauths"],
```

Comment:
Is it missing `authentication.config.openshift.io` and `authentication.operator.openshift.io`?

Reply:
Was focused on what we are driving into the cluster via hive.